infra/aws/terraform/modules/registry-k8s-io-s3-bucket/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "bucket_name" {
```

Comment:
nit: I think the CI checks will want to see the copyright header here as well

Reply:
header added.